tests/test_misc.py

```diff
@@ -1516,3 +1516,89 @@ def test_revault(revault_network, bitcoind, executor):
         wait_for(
             lambda: w.rpc.listvaults([], [deposit])["vaults"][0]["status"] == "canceled"
         )
+
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_reorged_revault(revault_network, bitcoind):
```

Comment:
Just a meta nit: if you can regroup it with the other reorg tests it would be nice :)

Reply:
Pushed it myself